rdr_service/genomic/genomic_job_components.py

```diff
@@ -695,14 +711,9 @@ def update_member_for_aw2(self, member: GenomicSetMember):
         :param member:
         """
 
+        state = GenomicWorkflowState.AW2
         member.aw2FileProcessedId = self.file_obj.id
-
-        # Only update the state if it was AW1
-        if member.genomicWorkflowState == GenomicWorkflowState.AW1:
-            member.genomicWorkflowState = GenomicWorkflowState.AW2
-            member.genomicWorkflowStateModifiedTime = clock.CLOCK.now()
-
-        self.member_dao.update(member)
```

Comment:
The logic here prevents state from being updated on re-ingestion if we need to process the file again for whatever reason. If member.genomicWorkflowState is downstream of AW2, for instance GEM_READY, and we need to re-ingest the file, without this logic the state would be reverted back to AW2.